sematic/ui/packages/common/src/pages/RunSearch/filters/OwnersFilterSection.tsx

```diff
@@ -58,14 +72,15 @@ const OwnersFilterSection = forwardRef<ResettableHandle, OwnersFilterSectionProp
         <Container>
             <FormGroup>
                 <StyledFormControlLabel control={<Checkbox
-                    checked={filters.has("current_user_id")}
-                    onChange={(e, checked) => toogleFilter("current_user_id", checked)} />} label="Your runs"
+                    checked={filters.has(currentUser!.id)}
+                    onChange={(e, checked) => toogleFilter(currentUser!.id, checked)} />} label="Your runs"
```

Comment:
Same comment everywhere. Here `e` is not even used.

Reply:
Sounds good. Fixed. Thanks!